tmcprototype/CspSubarrayLeafNode/CspSubarrayLeafNode/CspSubarrayLeafNode.py

```diff
@@ -80,7 +80,7 @@ def commandCallback(self, event):
     # Device Properties
     # -----------------
     CspSubarrayNodeFQDN = device_property(
-        dtype='str', default_value="mid-csp/elt/subarray01"
+        dtype='str', default_value="mid_csp/elt/subarray_01"
```

Comment:
is there a way to avoid hardcoded default value, as the same value will be used for initializing all 16 CSP subarrayleaf nodes?

Reply:
We have tried a way that we were extracting the SubarrayNode.Number(1,2,3......16) from SubarrayNode and append it with CspSubarray, CspSubarrayLeafNode and SdpSubarray FQDN to produce FQDN dynamically. But whenever we tried to do make test the device name also changed, because of that we can not get the SubarrayNode.Number and failed to produce correct FQDN.

Reply:
Use PROP_DEF_VAL_CSP_MID_SA1 variable from CONST.py file instead of hardcoded string in code.

Reply:
It has been changed as per the suggestions.
commit ID: [47c87a46eff5142110ead02b9377b5c29e5c655d](https://github.com/ska-telescope/tmc-prototype/pull/40/commits/47c87a46eff5142110ead02b9377b5c29e5c655d)